app/views/comment_mailer/comment_reply_sent_notification.html.erb

```diff
@@ -0,0 +1,33 @@
+<% content_for :message do %>
+
+  You replied to a comment on
+  <% if @comment.ultimate_parent.is_a?(Tag) %>
+    the tag
+    <%= style_link(@comment.ultimate_parent.commentable_name, {:controller => :tags, :action => :show, :id => @comment.ultimate_parent.to_param, :only_path => false}) %>:
```

Comment:
You should be able to use the `tag_path` helper for the link. Don't worry about the existing templates though.

Reply:
I've tried `tag_path(@comment.ultimate_parent)` and I get a 
```
 ActionView::Template::Error:
       undefined method `tag_path'
```
Is there anything I'm doing incorrectly?

Reply:
Sorry, it's probably that Rails doesn't allow relative links in mailers. Does `tag_url` works?

Reply:
Yup, thanks!